packages/account-utils/src/PasswordPolicyClass.ts

```diff
@@ -34,25 +74,108 @@ class PasswordPolicy {
 		this.throwError = throwError;
 	}
 
-	set forbidRepeatingCharactersCount(value) {
-		this._forbidRepeatingCharactersCount = value;
-		this.regex.forbiddingRepeatingCharacters = new RegExp(`(.)\\1{${this.forbidRepeatingCharactersCount},}`);
+	get passwordForbidRepeatingCharactersCount() {
+		return this.forbidRepeatingCharactersCount;
 	}
 
-	get forbidRepeatingCharactersCount() {
-		return this._forbidRepeatingCharactersCount;
+	set passwordForbidRepeatingCharactersCount(value: number) {
```

Comment:
I believe it would be better to remove all the `set`s and let the people use only through the constructor.

Reply:
Moved it to the constructor!